packages/apple-pay-integration/src/ApplePayPaymentMethod.tsx

```diff
@@ -16,10 +26,11 @@ const ApplePaymentMethod: FunctionComponent<PaymentMethodProps> = ({ method, che
             } catch (error) {
                 if (error instanceof Error) {
                     onUnhandledError(error);
-                }                
+                }
             }
         };
-        initializePayment();
+
+        void initializePayment();
```

Comment:
Just a feeling, we'll see `void` everywhere in the future, hah.

Reply:
Yes, we will be c sharp devs soon by the looks of it :D